src/core/trulens/core/database/migrations/versions/4_add_app_name_and_version_fields.py

```diff
@@ -0,0 +1,57 @@
+"""add app name and version fields
+
+Revision ID: 4
+Revises: 3
+Create Date: 2024-08-16 12:46:49.510690
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = "4"
+down_revision = "3"
+branch_labels = None
+depends_on = "1"
+
+
+def upgrade(config) -> None:
+    prefix = config.get_main_option("trulens.table_prefix")
+
+    if prefix is None:
+        raise RuntimeError("trulens.table_prefix is not set")
+
+    # TODO: The automatically generated code below likely references
+    #       tables such as "trulens_feedback_defs" or "trulens_records".
+    #       However, the common prefix for these tables "trulens_" is
+    #       actually configurable and so replace it with the variable
+    #       prefix.
+    #       e.g. replace "trulens_records" with prefix + "records".
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        prefix + "apps",
+        sa.Column("app_name", sa.VARCHAR(length=256), nullable=False),
+    )
+    op.add_column(
```

Comment:
Is there an existing column for app_metadata?

Reply:
kind of, `app_metadata` gets lumped with other `AppDefinition` attributes in the app_json. Do we want to make it its own column?